tests/CMakeLists.txt

```diff
@@ -4,10 +4,15 @@ project(models_test)
 include(CTest)
 enable_testing()
 
+if(NOT WIN32)
```

Comment:
I think it would be nice to include a quick comment here on why this is needed:

```suggestion
# The Catch CMake tools do not seem to work correctly on Windows.
if (NOT WIN32)
```